tests/fake_device/companion.py

```diff
@@ -57,6 +57,14 @@
 }
 
 
+class HidEvent:
```

Comment:
You can replace this with a dataclass.

Reply:
yes I know now what a dataclass is thanks :-)